paparazzi/paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -155,6 +164,9 @@ class PaparazziPlugin : Plugin<Project> {
 
         test.inputs.dir(mergeResourcesOutputDir)
         test.inputs.dir(mergeAssetsOutputDir)
+        test.inputs.files(nativePlatformFiles)
```

Comment:
I'd also add `.withPropertyName("paparazzi.layoutlibnative")` to name this input, so it shows up nicely in Gradle enterprise in case that is the cause for task being not up to date.

Reply:
done